ui/tests/e2e/test_config.ts

```diff
@@ -0,0 +1,12 @@
+export let SERVER_URL = `${
+  process.env.SERVER_URL || 'http://localhost:3000/'
+}#/`
```

Comment:
prefer to remove this trailing `/` and adding it to the routes in next line.

Reply:
Got, will update it soon.

Reply:
done!